xain_fl/grpc/coordinator.py

```diff
@@ -8,14 +8,15 @@
 import time
 from concurrent import futures
 from threading import Lock
-from typing import Dict, List, Optional, Tuple
+from typing import Dict, List, Tuple
 
 import grpc
+import numpy as np
 from google.protobuf.internal.python_message import GeneratedProtocolMessageType
 from numproto import ndarray_to_proto, proto_to_ndarray
-from numpy import ndarray
```

Comment:
As far as I can see we are using only ndarray from numpy package. Can we please bring this import back, instead of importing whole numpy?

Reply:
i'll change it back to importing only `ndarray` but basically everywhere else each module imports `numpy`, so it's inconsistent. to me it makes more sense to import `numpy` otherwise reading the code you don't know where this `ndarray` comes from. we can also import individual functions, it's fine, as long as we decide something and we are consistent.

Reply:
Totally get your point. In a private project before we just imported the functions and types, that are used, but since this is open source, we probably should think in the direction of how the user would like to interact with the code. On that point I agree with @wilk10 because most of the users will expect `np.something`. I know, that this is not leading into having a consistent way.

Reply:
Since we decided to continue with the most familiar approach for people, who are working with machine learning, this can be marked as resolved.